test/Content.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import Content from "../src/components/Content";
+
+const movieData = [{
+    "id": 337167,
+    "title": "Fifty Shades Freed",
+    "tagline": "Don't miss the climax",
+    "vote_average": 6.1,
+    "vote_count": 1195,
+    "release_date": "2018-02-07",
+    "poster_path": "https://image.tmdb.org/t/p/w500/3kcEGnYBHDeqmdYf8ZRbKdfmlUy.jpg",
+    "overview": "Believing they have left behind shadowy figures from their past, newlyweds Christian and Ana fully embrace an inextricable connection and shared life of luxury. But just as she steps into her role as Mrs. Grey and he relaxes into an unfamiliar stability, new threats could jeopardize their happy ending before it even begins.",
+    "budget": 55000000,
+    "revenue": 136906000,
+    "genres": [
+    "Drama",
+    "Romance"
+    ],
+    "runtime": 106
+}];
+
+describe('should render Content component', () => {
+    const tree = shallow(<Content movies={movieData} records={10} />);
+    test('Snapshot test', () => {
+        expect(tree).toMatchSnapshot();
+    });
+});
+
+test('should render Content component with records', () => {
+    const tree = mount(<Content movies={movieData} records={10} />);
+    expect(tree.prop('records')).toEqual(10);
```

Comment:
I'd suggest changing this test to check, if value provided in `records` prop is properly rendered by component.

Checking, if prop value is equal to provided value, tests if React works properly, not component.